goon_test.go

```diff
@@ -856,6 +869,49 @@ type MigrationB struct {
 	FinalField     string            `datastore:"final,noindex"`
 }
 
+func (m MigrationB) parent() *datastore.Key {
+	return m.Parent
+}
+
+func (m MigrationB) number() int32 {
+	return m.FancyNumber
+}
+
+// MigrationA with PropertyLoadSaver interface
+type MigrationPlsA MigrationA
+
+// MigrationB with PropertyLoadSaver interface
+type MigrationPlsB MigrationB
+
+func (m MigrationPlsA) parent() *datastore.Key {
+	return m.Parent
+}
+func (m MigrationPlsA) number() int32 {
+	return m.Number
+}
+func (m *MigrationPlsA) Save() ([]datastore.Property, error) {
+	return datastore.SaveStruct(m)
+}
+func (m *MigrationPlsA) Load(props []datastore.Property) error {
+	return datastore.LoadStruct(m, props)
+}
+
+func (m MigrationPlsB) parent() *datastore.Key {
+	return m.Parent
+}
+func (m MigrationPlsB) number() int32 {
+	return m.FancyNumber
+}
+func (m *MigrationPlsB) Save() ([]datastore.Property, error) {
+	return datastore.SaveStruct(m)
+}
+func (m *MigrationPlsB) Load(props []datastore.Property) error {
+	return datastore.LoadStruct(m, props)
+}
+
+// Make sure that MigrationC implements datastore.PropertyLoadSaver
+var _, _ datastore.PropertyLoadSaver = &MigrationPlsA{}, &MigrationPlsB{}
```

Comment:
The `MigrationC` reference in the comment seems outdated. Should change the wording to something like: _Make sure that these implement datastore.PropertyLoadSaver_.

Reply:
Fixed dd18dfdcf507a3c9df03d209d940e71b82833e10